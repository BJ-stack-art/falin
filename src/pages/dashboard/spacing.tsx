import { CircularProgress } from "@mui/material"
import React, { Suspense, lazy } from "react"
const SpacingGuide = lazy(() => import("@/features/dashboard/spacing"))

const SpacingPage: React.FC = () => {
  return (
    <Suspense fallback={<CircularProgress />}>
      <SpacingGuide />
    </Suspense>
  )
}

export default SpacingPage
