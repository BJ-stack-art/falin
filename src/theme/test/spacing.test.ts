import { describe, expect, test } from "vitest"
import { spacing } from "../spacing"

describe("Theme Spacing", () => {
  test("should match snapshot", () => {
    expect(spacing).toMatchSnapshot()
  })
})
