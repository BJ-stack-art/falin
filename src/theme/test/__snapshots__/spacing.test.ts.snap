// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Theme Spacing > should match snapshot 1`] = `
[
  0,
  4,
  8,
  12,
  16,
  20,
  24,
  32,
  40,
  48,
  64,
  80,
  96,
  128,
  160,
  192,
  224,
  256,
]
`;
